SEP-0006.md

```diff
@@ -28,14 +28,14 @@ table lists the start and end of individual appointments.
 
 | Name                          | Start       |  Expiration |
 |-------------------------------|-------------|-------------|
-| Monica Bobra                  | 6 Mar 2019  | 6 Mar 2021  |
-| Bin Chen                      | 2 Aug 2019  | 6 Mar 2021  |
+| Monica Bobra                  | 6 Mar 2019  | 19 May 2022  |
+| Bin Chen                      | 2 Aug 2019  | 19 May 2022  |
 | Steven Christe                | 6 Mar 2019  | 9 Mar 2022  |
```

Comment:
why not make this 19 Mar as well to be consistent with the other two expiration dates in March 2022?

Reply:
Sorry, forgot to type the "1". Thanks for catching it!